project.code-workspace

```diff
@@ -193,10 +193,7 @@
       "titleBar.activeForeground": "#e7e7e7",
       "titleBar.inactiveBackground": "#eb542499",
       "titleBar.inactiveForeground": "#e7e7e799",
```

Comment:
I think we should remove this from the commit

Reply:
I think this came in as part of the rebase I did.